setup/templates/yum/post-actions/tomcat.action

```diff
@@ -0,0 +1,2 @@
+tomcat*:install:"$SCRIPTS_PATH/config-tomcat-security.sh"
```

Comment:
how it will know the value of $SCRIPT_PATH here?

Reply:
Will be resolved through envsubst (check aws-setup-web-server.sh)